client/components/Recipe.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { useParams } from 'react-router-dom'
+import { useSelector } from 'react-redux'
+
+export default function Recipe() {
+  const { id } = useParams()
+  const recipes = useSelector((state) => state.recipes)
+  const recipe = recipes[id]
+
+  const label = recipe.recipe.label
```

Comment:
All of these could be simplified into one line
```suggestion
  const { label, images, recipe, healthLabels } = recipe.recipe
```
Just saves some lines

Reply:
Completed, thank you.